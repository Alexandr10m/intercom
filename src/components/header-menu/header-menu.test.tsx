import * as React from "react";
import * as renderer from "react-test-renderer";
import HeaderMenu from "./header-menu";


describe(`HeaderMenu Snapshots`, () => {
  it(`shoult render correctly`, () => {
    const tree = renderer
      .create(<HeaderMenu/>)
      .toJSON();

      expect(tree).toMatchSnapshot();
  });
});
