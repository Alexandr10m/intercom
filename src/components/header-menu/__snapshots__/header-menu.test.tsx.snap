// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderMenu Snapshots shoult render correctly 1`] = `
<div
  className="nav-container"
>
  <nav
    className="nav-bar"
  >
    <a
      className="nav-bar__logo-link"
    >
      <img
        alt="логотип"
        src="..\\\\..\\\\images\\\\logo.png"
      />
    </a>
    <ul
      className="menu nav-bar__menu"
    >
      <li
        className="menu__item"
      >
        <a
          className="menu_link"
          href="#profile"
        >
          Работы
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu_link"
          href="#services"
        >
          Услуги
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu_link"
          href="#advantages"
        >
          Преимущества
        </a>
      </li>
      <li
        className="menu__item"
      >
        <a
          className="menu_link"
          href="#team"
        >
          Команда
        </a>
      </li>
    </ul>
  </nav>
  <a
    className="header__phone-link"
    href="tel:+380676297972"
  >
    + 380 (67) 629 79 72
  </a>
</div>
`;
